packages/@lwc/errors/src/__tests__/errors.spec.ts

```diff
@@ -75,4 +77,22 @@ describe('error validation', () => {
 
         traverseErrorInfo(CompilerErrors, checkUniqueness, 'compiler');
     });
+
+    it('Next error code is updated', () => {
+        const errorCodes: number[] = [];
+        traverseErrorInfo(
+            CompilerErrors,
+            (error) => {
+                errorCodes.push(error.code);
+            },
+            'compiler'
+        );
+        const expectedNextErrorCode = 1 + Math.max(...errorCodes);
```

Comment:
a different approach to achieve this, may be (either way is fine):

```suggestion
        let lastErrorCode = 1001;
        traverseErrorInfo(
            CompilerErrors,
            (error) => {
                lastErrorCode = Math.max(lastErrorCode, error.code);
            },
            'compiler'
        );
        const expectedNextErrorCode = 1 + lastErrorCode;
```

Reply:
Sure, this is fine!